php/commands/site.php

```diff
@@ -207,6 +207,9 @@ public function connection_mode($args, $assoc_args) {
       $action = 'set';
     }
     $env = Input::env($assoc_args, 'env');
+    if (($env == 'test' || $env == 'live') && $action == 'set') {
+      Terminus::error("Connection mode cannot be set in Test or Live environments");
```

Comment:
I thought of that, and it was the first direction I headed. I figured moving the check up a few lines would help it to fail faster. In the grand scheme of things, there's probably not much speed to be gained over the course of a dozen lines. I'll try moving it down a little later tonight. Thanks!